sdk/identity/identity/images/mermaidjs/DefaultAzureCredentialAuthFlow.md

```diff
@@ -0,0 +1,28 @@
+```mermaid
+%% STEPS TO GENERATE IMAGE
+%% =======================
+%% 1. Install mermaid CLI (see https://github.com/mermaid-js/mermaid-cli/blob/master/README.md)
+%% 2. Run command: mmdc -i DefaultAzureCredentialAuthFlow.md -o DefaultAzureCredentialAuthFlow.svg
+
+flowchart LR;
+    A(Environment):::deployed ==> B(Managed Identity):::deployed ==> C(VS Code):::developer ==> D(Azure CLI):::developer ==> E(Azure PowerShell):::developer;
+
+    subgraph CREDENTIAL TYPES;
+        direction LR;
+        Deployed(Deployed service):::deployed ==> Developer(Developer):::developer;
```

Comment:
```suggestion
        Deployed(Deployed Service):::deployed ==> Developer(Developer):::developer;
```
(nit)
“Deployed service” is the only sentence (or so) that doesn’t have every word with the first letter in upper case.

Reply:
Let's stick with sentence casing here because "service" isn't a proper noun in this context.